test/test.js

```diff
@@ -13,5 +14,14 @@ exports.babel = {
 		test.deepEqual(json.file, 'fixture-compiled.js');
 		test.ok(/\/\/# sourceMappingURL=fixture-compiled.js.map\n$/.test(code));
 		test.done();
+	},
+	transpile: function (test) {
+		var res = babel.transformFileSync('test/fixture.js', {
+			presets: ['babel-preset-es2015']
+		});
```

Comment:
I can remove this test but I think updating the grunt file should sufficient then.